src/lib.rs

```diff
@@ -225,14 +224,6 @@ impl ToString for Level {
 /// Builder for a generic request to Rollbar.
 pub struct ReportBuilder<'a> {
     client: &'a Client,
-    send_strategy: Option<
-        Box<
-            dyn Fn(
-                Arc<hyper::Client<hyper_tls::HttpsConnector<hyper::client::HttpConnector>>>,
-                String,
-            ) -> thread::JoinHandle<Option<ResponseStatus>>,
-        >,
-    >,
```

Comment:
i ripped out the send strategy feature, it made it hard to change the send return type from thread join handle to option response.